runtime/src/prioritization_fee.rs

```diff
@@ -0,0 +1,358 @@
+use {
+    crate::transaction_priority_details::GetTransactionPriorityDetails,
+    solana_measure::measure,
+    solana_sdk::{
+        clock::Slot,
+        pubkey::Pubkey,
+        saturating_add_assign,
+        transaction::{SanitizedTransaction, MAX_TX_ACCOUNT_LOCKS},
+    },
+    std::collections::HashMap,
+};
+
+#[derive(Debug, Default)]
+struct PrioritizationFeeMetrics {
+    // Count of writable accounts in slot; including accounts have minimum prioritization fee equal
+    // to blocker's minimum fee, the number of these irrelevant accounts is the difference between
```

Comment:
I actually found that this comment made things a little more confusing for me. I think "Count of all writable accounts in the slot" is sufficient, and then the comment for relevant_writable_accounts_count is very clear and makes the difference obvious.

Reply:
Please move comments like this to Tao's original PR, and only consider the last 3 commits of this PR here